### PR TITLE
main: add interval flag for testing

### PR DESCRIPTION
**Pull request by xiang90**
/cc @hongchaodeng

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -16,32 +16,33 @@ import (
...
 	modeEndpointsCheckpoint = "endpoints"
 	modeIptablesCheckpoint  = "iptables"
 
 	dirperm = 0700
 
-	defaultVIP           = "10.3.0.15"
-	defaultCheckpointDir = "/etc/kubernetes/selfhosted-etcd"
+	defaultVIP            = "10.3.0.15"
+	defaultCheckpointDir  = "/etc/kubernetes/selfhosted-etcd"
+	defaultClusterInteval = 30 * time.Second
```
Comment (hongchaodeng) [inline main.go]: Why not 1 min?

**Comment by hongchaodeng** [comment]
lgtm in general

File: main.go (hunk 1 above)
Comment (xiang90) [inline main.go]: i do not know what the best interval should be now. actually it is the shorter the better. a duration that is larger than 5seconds should be OK anyway. but let's try 30 seconds first.

**Comment by hongchaodeng** [comment]
lgtm

**Pull request merged**